terraso_backend/apps/graphql/schema/memberships.py

```diff
@@ -100,6 +101,7 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
 
         if group.membership_type == Group.MEMBERSHIP_TYPE_CLOSED:
             membership.membership_status = Membership.PENDING
+            EmailNotification.SendMembershipRequest(user, group)
```

Comment:
Suggestion: Can we import and call the function directly? What is the use for the `EmailNotification` class?

Reply:
I think that if the only thing that the class is doing is proving more redability I think we should improve the name of the functions, also a suggestion I don't think we have more usages of static functions anywhere in the code, but I can be wrong